apps/lead/migrations/0037_auto_20210715_0432.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2 on 2021-07-15 04:32
+
+from django.db import migrations
+
+
+def set_lead_is_assessment_(Lead, Assessment):
+    assessments = Assessment.objects.all()
+    leads = Lead.objects.filter(
+        assessment__in=assessments
+    )
+    for lead in leads:
+        lead.is_assessment_lead = True
+        lead.save(update_fields=['is_assessment_lead'])
```

Comment:
We should be able to use just .update() here.

Also, let's use `_set_lead_is_assessment`